### PR TITLE
Bumped Heapster to v1.2.0-beta.3

### PR DESCRIPTION
**Pull request by piosz**
The new version contains Kubernetes deps in version `1.4.0-beta.0`

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/32362)

### CODE REVIEW

**Comment by piosz** [comment]
cc @mwielgus

File: cluster/addons/cluster-monitoring/standalone/heapster-controller.yaml [hunk 1]
```diff
@@ -83,7 +83,7 @@ spec:
             - --memory={{ base_metrics_memory }}
             - --extra-memory={{ metrics_memory_per_node }}Mi
             - --threshold=5
-            - --deployment=heapster-v1.2.0-beta.2
+            - --deployment=heapster-v1.2.0-beta.3
```
Comment (wojtek-t) [inline cluster/addons/cluster-monitoring/standalone/heapster-controller.yaml]: Can you please also bump it in test/kubemark/resources/heapster_template.json ?

Comment (piosz) [inline cluster/addons/cluster-monitoring/standalone/heapster-controller.yaml]: Done. I need to check it

**Comment by wojtek-t** [comment]
LGTM - thanks!

**Comment by piosz** [comment]
@k8s-bot kubemark test this issue: #IGNORE

**Comment by piosz** [comment]
@k8s-bot e2e test this issue: #IGNORE

**Comment by piosz** [comment]
@k8s-bot test this issue: #IGNORE

**Comment by piosz** [comment]
No significant changes since `v1.2.0-beta.2`. Mostly updated Kubernetes deps to the latest release (1.4.0-beta.0).

**Comment by pwittrock** [comment]
1.  Please provide a risk assessment in the description
  - Risk surface of accepting this cherrypick: what could go wrong
  - How easy it is to revert if needed 
  - Why it is worth the risk, burden to cherrypick and verify the tests stay green, and noise it creates

2. How urgent is this?  blocking / important / best effort?  We have dozens of cherrypick requests and each one creates a lot of overhead because large batches introduce a high-risk of destabilizing the tests in a way that is hard to track down.   We have to break them up into smaller batches and monitor the tests to catch issues early on and track them back to their root cause.

**Comment by piosz** [comment]
Low risk change - not much changes. It's easy to revert (just Heapster manifest files being touched).

We target Heapster 1.2 to be released together with Kubernetes 1.4. This is a next beta release of Heapster (hopefully the last one - if the passes we will release v1.2.0).

It may broke HPA and Monitoring e2e tests.

**Pull request merged**